crates/torii/core/src/engine.rs

```diff
@@ -18,6 +18,9 @@ use tracing::{error, info, trace, warn};
 
 use crate::processors::{BlockProcessor, EventProcessor, TransactionProcessor};
 use crate::sql::Sql;
+
+type HasWorldEvent = bool;
```

Comment:
May not need this one as we're testing booleans in the match, would be great to have in the function doc instead. 👍